strapi/strapi
# [CTB] Add format examples in date field select

### What does it do?

It adds format examples in date field select:

<img width="886" alt="Screenshot 2022-03-17 at 09 14 14" src="https://user-images.githubusercontent.com/13311463/158765672-50833d4c-c62f-40eb-8935-2ee1125da466.png">


### Why is it needed?

So users have information about the format they are choosing.

### How to test it?

1. Go to the CTB 
2. Click on add new field and choose the Date one
3. See the date format example
